features/step_definitions/generator_steps.rb

```diff
@@ -28,6 +35,11 @@
   EOH
 end
 
+Given(/^I debug$/) do
```

Comment:
`pry` isn't a dep, it's only in the `Gemfile`. `gem install omnibus` won't install it.